### PR TITLE
add feature flag vaosv2next

### PR DESCRIPTION
**Pull request by jenniemc**
## Description
add the feature flag `va_online_scheduling_vaosv2next`

## Original issue(s)
department-of-veterans-affairs/va.gov-team#44559

## Testing done
n/a

## Screenshots
n/a

## Acceptance criteria
- [ ] feature flag is ready for use in development
- [ ] [Documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/product/feature_toggles.md) is updated

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by theodur — APPROVED** [review]

**Review by simiadebowale — COMMENTED** [review]
LGTM!

**Comment by jenniemc** [comment]
> LGTM!

Hi @simiadebowale , can you select the approve radio button in the review page? Looks like this ticket needs 2 approvals; one from the Platform Release Team and another from VAOS FE Team. Thanks!

**Comment by simiadebowale** [comment]
> > LGTM!
> 
> Hi @simiadebowale , can you select the approve radio button in the review page? Looks like this ticket needs 2 approvals; one from the Platform Release Team and another from VAOS FE Team. Thanks!

Haha, I thought I selected the radio button. Doing that now.

**Review by simiadebowale — APPROVED** [review]

**Pull request merged**
